src/apis/platformvm/api.ts

```diff
@@ -1577,7 +1577,7 @@ export class PlatformVMAPI extends JRPCAPI {
       )
     }
 
-    const builtUnsignedTx: UnsignedTx = utxoset.buildAddDelegatorTx(
+    const builtUnsignedTx: UnsignedTx = utxoset.buildAddDelegatorTxx(
```

Comment:
What is the intent of changing everything related to `DelegatorTx` to `DelegatorTxx`?

Reply:
Delegating will be replaced by Deposit with a different logic, just wanted to leave it in to better find code places.
I'll rename it to Deposit, then it makes things clearer.